README.md

```diff
@@ -6,31 +6,31 @@
 [![OpenCollective](https://opencollective.com/phalcon/backers/badge.svg)](#backers)
 [![OpenCollective](https://opencollective.com/phalcon/sponsors/badge.svg)](#sponsors)
 
-Phalcon is an open source web framework delivered as a C extension for the PHP language providing high performance and lower resource consumption.
+Phalcon is an open-source web framework delivered as a C extension for the PHP language providing high performance and lower resource consumption.
 
 A big thank you to [our Backers](https://github.com/phalcon/cphalcon/blob/master/BACKERS.md); you rock!
 
 ## Getting Started
 
 Phalcon is written in [Zephir/C](https://zephir-lang.com/) with platform independence in mind.
-As a result, Phalcon is available on Microsoft Windows, GNU/Linux, FreeBSD and MacOS.
-You can either download a binary package for the system of your choice or build it from source.
+As a result, Phalcon is available on Microsoft Windows, GNU/Linux, FreeBSD, and macOS.
+You can either download a binary package for the system of your choice or build it from the source.
 
 **NOTE:** Phalcon requires the [PSR PHP extension](https://github.com/jbboehr/php-psr) to be installed and enabled.
 
 ## Installation
 
-For detailed installation instructions you can check our [installation](https://docs.phalcon.io/4.0/en/installation) page in the docs.
+For detailed installation instructions, you can check our [installation](https://docs.phalcon.io/4.0/en/installation) page in the docs.
 
 ## Generating API Documentation
 
-Generating new documentation files for docs repository can be done using the script in tests/_ci/generate-api-docs.php.
+Generating new documentation files for docs repository can be done using the script in tests/_ci/generate-API-docs.php.
 Steps:
 - Clone the phalcon repo
-- Checkout the tag you would like to generate docs for.
-- Run `php tests/_ci/generate-api-docs.php`
+- Check out the tag you would like to generate docs for.
+- Run `PHP tests/_ci/generate-API-docs.php`
```

Comment:
Same here. `php` is the command not uppercase. Same with `API`

Reply:
Yep agree with @niden on this. Second line needs to be all lowercase